src/output/icons.rs

```diff
@@ -58,7 +59,7 @@ impl Icons {
     const LANG_KOTLIN: char     = '\u{e634}';  // 
     const LANG_OCAML: char      = '\u{e67a}';  // 
     const LANG_PERL: char       = '\u{e67e}';  // 
-    const LANG_PHP: char        = '\u{e73d}';  // 
```

Comment:
PHP does not use the elephant as the current logo.

Reply:
It might not be official, but the elephant mascot is well-known and appears when you Google image search "php". I personally prefer the PHP elephant, and I know others do, too. Maybe if someone else says they don't want this, I can change it. Other projects such as `lsd`, [ranger_devicons](https://github.com/alexanderjeurissen/ranger_devicons) (I've contributed to both), [vim-devicons](https://github.com/ryanoasis/vim-devicons/tree/master) and [all-the-icons.el for Emacs](https://github.com/domtronn/all-the-icons.el/blob/master/logo.png) use the elephant for PHP, so it is not unheard of, and it would make `eza` more consistent with these other projects.